### ISSUE TITLE
Add git  and github documentation in the course

### ISSUE THREAD

**Issue by AdityaSrivastava185** [AdityaSrivastava185/Reactjs-For-Beginners#21]
The course does not have the clear explaination of the git and github topic , and the link is also not redirecting to the page , please add the documentation on git and github for the beginners to understand the concepts of the git and github . A clear and concise explaination on git and github is required from the beggining to the advance concepts

**Comment by vishalranaut** [comment]
Add Comprehensive Documentation on Git and GitHub done

**Comment by favourachara07** [comment]
Can I please be assigned to this issue?

**Comment by yashksaini-coder** [comment]
Hi, @AdityaSrivastava185 can i also collaborate on resolving this issue?

**Comment by AdityaSrivastava185** [comment]
> Hi, @AdityaSrivastava185 can i also collaborate on resolving this issue?

Sure @yashksaini-coder  you also can collaborate on this issue , just implement the changes and make a PR

**Comment by AdityaSrivastava185** [comment]
> Can I please be assigned to this issue?

Hi @favourachara07 you can improve the issue and then make a PR , for now it is open and collaborative to everyone

**Comment by favourachara07** [comment]
Hey @yashksaini-coder 
I would like to work on the topic, git if that's okay with you

**Comment by AdityaSrivastava185** [comment]
> Hey @yashksaini-coder I would like to work on the topic, git if that's okay with you

That's absolutely alright 🙂 @favourachara07

**Comment by favourachara07** [comment]
Sorry for asking this but I’m a new to open source 
Should i create a branch separate from @yashksaini-coder or we will share a single branch for the issue

**Comment by AdityaSrivastava185** [comment]
> Sorry for asking this but I’m new to open source Should i create a branch separate from @yashksaini-coder or we will share a single branch for the issue

Yes , You should create a new branch then make changes and make a PR , i suggest to go through [Contributing-Guidelines](https://github.com/AdityaSrivastava185/Reactjs-For-Beginners/blob/main/CONTRIBUTING.md) before making changes and making a PR

**Comment by favourachara07** [comment]
Thank you

**Comment by AdityaSrivastava185** [comment]
> Thank you

Happy to help you , Happy Contribution ✨

**Comment by yashksaini-coder** [comment]
Okh I will create a new branch that will contain the beginners level guide of GitHub. It will be a complete different section. For which I will be raising an entire new issue. That way @favourachara07  and I both can tackle the problems very easily while also following the contribution guidelines. 

Is this okay for you... ?
@favourachara07 
@AdityaSrivastava185

**Comment by AdityaSrivastava185** [comment]
> Okh I will create a new branch that will contain the beginners level guide of GitHub. It will be a complete different section. For which I will be raising an entire new issue. That way @favourachara07 and I both can tackle the problems very easily while also following the contribution guidelines.
> 
> Is this okay for you... ? @favourachara07 @AdityaSrivastava185

Yes 👍

**Comment by yashksaini-coder** [comment]
Good then. We will proceed this way.

**Comment by AdityaSrivastava185** [comment]
Great Work @favourachara07 , Merged this issue but still there are lot of changes to improve this , So keep contributing

**Comment by favourachara07** [comment]
Please could you tell me where i can make changes  so that i can know and make improvements 
Thank you

**Comment by AdityaSrivastava185** [comment]
@favourachara07  , currently you can add multiple images , also some git and github working concepts to  , multiple github programs and perks , make the documentataion clean and easy to learn step by step for those people who wants to learn on the way and implement it

**Comment by favourachara07** [comment]
okay 
so i should make these changes in my branch
then i will make a pr like before

**Comment by AdityaSrivastava185** [comment]
Yes

**Comment by yadavmangesh07** [comment]
Here is a message you can use to ask the repository owner to assign the issue to you:

---

Hi @AdityaSrivastava185,

Could you please assign this issue to me? I would like to work on it.

Thank you!

Best regards,  
@yadavmangesh07
